### PR TITLE
Fix linter docs warnings

### PR DESCRIPTION
**Pull request by karimnaaji**
- Allow to run the linter without warning, housekeeping step prior to introducing more docs changes during this sprint
- Add a few missing `@returns` tags from both internal and public facing docs
- Cleanup some obsolete and leftover docs not associated to any code

### CODE REVIEW

File: src/terrain/elevation.js [hunk 1]
```diff
@@ -40,7 +41,7 @@ export class Elevation {
      * Altitude above sea level in meters at specified point.
      * @param {MercatorCoordinate} point Mercator coordinate of the point.
      * @param {number} defaultIfNotLoaded Value that is returned if the dem tile of the provided point is not loaded
-     * @param exaggerated
+     * @param exaggerated Whether to apply the styling exaggeration to the resulting elevation
```
Comment (domlet) [inline src/terrain/elevation.js:44]:
```suggestion
     * @param {boolean} `true` if styling exaggeration should be applied to the resulting elevation.
```
Not sure if this change is valid – can we improve the specificity of this message by adding a type?

File: src/terrain/elevation.js [hunk 2]
```diff
@@ -29,8 +29,9 @@ export type ElevationQueryOptions = {
 export class Elevation {
     /**
      * Helper around `getAtPoint` that guarantees that a numeric value is returned.
-     * @param point
-     * @param defaultIfNotLoaded
+     * @param {MercatorCoordinate} point Mercator coordinate of the point.
+     * @param {number} defaultIfNotLoaded Value that is returned if the dem tile of the provided point is not loaded
```
Comment (domlet) [inline src/terrain/elevation.js:33]:
```suggestion
     * @param {number} defaultIfNotLoaded Value that is returned if the dem tile of the provided point is not loaded.
```

Comment (domlet) [inline src/terrain/elevation.js:33]:
```suggestion
     * @param {number} defaultIfNotLoaded Value that is returned if the dem tile of the provided point is not loaded.
```

File: src/ui/handler/shim/touch_zoom_rotate.js [hunk 3]
```diff
@@ -77,7 +77,7 @@ export default class TouchZoomRotateHandler {
     /**
      * Returns true if the handler is enabled and has detected the start of a zoom/rotate gesture.
      *
-     * @returns {boolean}
+     * @returns {boolean} `true` if enabled a zoom/rotate gesture was detected.
```
Comment (domlet) [inline src/ui/handler/shim/touch_zoom_rotate.js:80]:
```suggestion
     * @returns {boolean} `true` if enabled and a zoom/rotate gesture was detected.
```
^ Added "and" to connect the two requirements.

**Review by domlet — APPROVED** [review]
Thank you @karimnaaji for this careful housekeeping, much appreciated.

File: src/terrain/elevation.js (hunk 1 above)
Comment (karimnaaji) [reply src/terrain/elevation.js:44]: Yep, good catch

File: src/terrain/elevation.js [hunk 4]
```diff
@@ -40,7 +41,7 @@ export class Elevation {
      * Altitude above sea level in meters at specified point.
      * @param {MercatorCoordinate} point Mercator coordinate of the point.
      * @param {number} defaultIfNotLoaded Value that is returned if the dem tile of the provided point is not loaded
-     * @param exaggerated
+     * @param {boolean} `true` if styling exaggeration should be applied to the resulting elevation.
```
Comment (karimnaaji) [inline src/terrain/elevation.js:44]:
```suggestion
     * @param {boolean} exaggerated `true` if styling exaggeration should be applied to the resulting elevation.
```

**Pull request merged**